manage_breast_screening/notifications/management/commands/retry_failed_message_batch.py

```diff
@@ -57,6 +58,21 @@ def handle(self, *args, **options):
                         message_batch=message_batch, response_json=response.json()
                     )
                 else:
+                    if response.status_code in RECOVERABLE_STATUS_CODES:
```

Comment:
@steventux 's was your suggestion around this to reuse the `mark_batch_as_failed` logic we already have in `command_helpers`? 
I think if we modified the `mark_batch_as_failed` method to take a `retry_count` then it would do what we want it to do